structuredefinitions/Extension-England-BusinessChangeTypedDateTime.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-BusinessChangeTypedDateTime" />
```

Comment:
This is stating 'Typed' whereas in the other assets it is 'Type', is this a typo? If so, update file name, id, url etc.